src/main/java/mate/academy/dao/impl/ShoppingCartDaoImpl.java

```diff
@@ -37,16 +38,16 @@ public ShoppingCart add(ShoppingCart shoppingCart) {
     @Override
     public ShoppingCart getByUser(User user) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
-            Query<ShoppingCart> query = session.createQuery("FROM ShoppingCart sc "
-                    + "LEFT JOIN FETCH sc.tickets t "
-                    + "LEFT JOIN FETCH t.movieSession ms "
-                    + "LEFT JOIN FETCH ms.movie "
-                    + "LEFT JOIN FETCH ms.cinemaHall "
-                    + "WHERE sc.user =:user", ShoppingCart.class);
+            Query<ShoppingCart> query
+                    = session.createQuery("from ShoppingCart s "
+                    + "left join fetch s.tickets t "
```

Comment:
Why did you rewrite it?
It is better to write all SQL commands in upper case, fix everywhere